db_test.go

```diff
@@ -1,19 +1,99 @@
 package main
 
 import (
-  "fmt"
+  "gopkg.in/ory-am/dockertest.v2"
+  "labix.org/v2/mgo"
+  "time"
+  
   "testing"
   
   "github.com/stretchr/testify/assert"
-  )
+)
 
 func TestLoadTemplate(t *testing.T) {
-  templates := getTemplatesByName("artikkel")  // We know this is in the db
+  var db *mgo.Session
+  c, err := dockertest.ConnectToMongoDB(15, time.Millisecond  * 500, func(url string) bool {
+    // Check if the docker image responds
+    var err error
+    db, err = mgo.Dial(url)
+    if err != nil {
+      return false
+    }
+    
+    // Check if the docker image responds to ping
+    return db.Ping() == nil
+  })
   
-  if (len(templates) > 1) {
-    // wtf
-  } else {
-    fmt.Printf("%v", templates)
-    assert.Equal(t, "artikkel", "templates[0][\"name\"]")
+  if err != nil {
+    // log.Fatalf("Could not connect to database: %s", err)
```

Comment:
Uncommenting this, and adding "log" to the list of imports makes this work again :)